api/app/signals/apps/api/urls_v1.py

```diff
@@ -3,8 +3,8 @@
 from django.conf import settings
```

Comment:
Since these URLs are all included under `/v1/` these are still version 1 URLs and I would like to keep the naming for now. I think we may want to look at the usage of `NameSpaceVersioning` in our project (it has been a source of bugs) and we should clean-up the `users` app which still has the v1 nesting in source files. (Cleaning-up `users` is out of scope of this PR's ticket.)